mpact/services.py

```diff
@@ -201,27 +199,16 @@ async def get_dialog(phone):
         return NOT_AUTHORIZED
 
 
-async def get_individual_msg(phone, individual_id, limit, offset):
+async def get_individual_msg(phone, individual_id):
     """
     Returns private chat messages if the user is authorized
     """
     async with client_context(phone) as client:
         if await client.is_user_authorized():
-            if limit and offset:
-                data = Message.objects.filter(individual=individual_id).order_by(
-                    "-date"
-                )[int(offset) : int(offset) + int(limit)]
-            elif limit:
-                data = Message.objects.filter(individual=individual_id).order_by(
-                    "-date"
-                )[: int(limit)]
-            else:
-                data = Message.objects.filter(individual=individual_id).order_by(
-                    "-date"
-                )
+            data = Message.objects.filter(individual=individual_id)
```

Comment:
Is this going to be a problem if there are a lot of messages with an individuals? ... How many is "a lot"? 1000? 10,000? Maybe this is not a reasonable concern?